presto-native-tests/src/test/java/com.facebook.presto.nativetests/TestAggregations.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativetests;
+
+import com.facebook.presto.nativeworker.NativeQueryRunnerUtils;
+import com.facebook.presto.nativeworker.PrestoNativeQueryRunnerUtils;
+import com.facebook.presto.testing.QueryRunner;
+
+public class TestAggregations
+        extends AbstractTestAggregationsNative
+{
+    private static final String storageFormat = "PARQUET";
```

Comment:
This is just testing parquet

Reply:
Yes the intention was to primarily test with parquet files. Most of the current e2e tests in `presto-native-execution` test with DWRF format. We can extend this to test with other file formats going forward.